dev/com.ibm.ws.microprofile.rest.client_fat/bnd.bnd

```diff
@@ -28,56 +28,57 @@ src: \
   test-applications/hostnameVerifierApp/src,\
   test-applications/jsonbContextApp/src,\
   test-applications/multiClientCdiApp/src,\
+  test-applications/MyRestClient/src,\
```

Comment:
It is still going to fail to build on linux because the directory name is still RESTClient

Reply:
directory name changed back when I rebased, fixed.... again...